packages/node/src/protocol/utils/get-outcome-increments.ts

```diff
@@ -142,12 +143,21 @@ function handleTwoPartyFixedOutcome(
 }
 
 function handleMultiAssetMultiPartyCoinTransfer(
-  // @ts-ignore
   encodedOutcome: string,
-  // @ts-ignore
   interpreterParams: MultiAssetMultiPartyCoinTransferInterpreterParams
 ): TokenIndexedCoinTransferMap {
-  throw new Error("UnimplementedError");
+  const decodedTransfers = decodeMultiAssetMultiPartyCoinTransfer(
+    encodedOutcome
+  );
+
+  return interpreterParams.tokenAddresses.reduce(
+    (tokenIndexedTransfers, tokenAddress: string, index: number) => {
```

Comment:
Rename accumulator `tokenIndexedTransfers` to `tokenIndexedCoinTransferMap`

Reply:
Done